ui/component/searchScreenInput/view.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+const os = require('os').type();
+
+const SearchScreenInput = () => {
+  const [searchString, setSearchSring] = useState('');
+  const inputRef = useRef();
+
+  useEffect(() => {
+    // refocus with ctrl-f after search
+    const handleKeyPress = (e) => {
+      if ((os !== 'Darwin' && e.ctrlKey && e.keyCode === 70) || (e.keyCode === 70 && e.metaKey)) {
+        window.focus();
+        inputRef.current.focus();
+        e.preventDefault();
+      }
+    };
+
+    window.addEventListener('keydown', handleKeyPress);
+    return () => window.removeEventListener('keydown', handleKeyPress);
+  }, [searchString]);
+
+  return (
+    <input
+      ref={inputRef}
+      onKeyPress={(e) => e.key === 'Enter' && window.find(searchString)}
```

Comment:
To match Chromes behavior `window.find` should be called with every key press, not just when you click enter. This code can be combined with `onChange`

Reply:
Hello @seanyesmunt, I have fixed all the request's for this PR except for this one, the reason why I did it this way in the first place is that electron has a weird behavior when using the `webContents.findInPage` method. Basically every time it is called, the input loses focus and I have been unable to fix this on my end. I really would like to finish implementing this feature, so If you have any advice I will take it, or I can implement it as I previously did if you allow me.

Reply:
I have re-uploaded the PR with all the other comments fixed, except for this. I gave it a try once again and I was unable to get the desired behavior with the webContents.findInPage method.

Reply:
@seanyesmunt Am I allowed to install external libraries? found one that might be of help with the misbehavior that I'm facing.

Reply:
I think that's fine. Also, check out some of the comments in https://github.com/lbryio/lbry-desktop/issues/5877 for other feedback.

Reply:
> I think that's fine. Also, check out some of the comments in #5877 for other feedback.

Mhh... Interesting seems that we are facing the same issues.